waku/v2/protocol/waku_filter/waku_filter_types.nim

```diff
@@ -45,7 +46,7 @@ type
 
   WakuFilter* = ref object of LPProtocol
     rng*: ref BrHmacDrbgContext
-    switch*: Switch
+    peerManager*: PeerManager
```

Comment:
It'd be good if we made this change consistently in other protocols, though I assume that's the plan. Also worth notifying Stimbus so they are aware of this change.

Oh, and CHANGELOG.md please!

Reply:
Yep, that is indeed the plan. Will inform Stimbus as well.

Oh, and of course - CHANGELOG! 🤦 Done in 4e7f068